package.json

```diff
@@ -38,10 +38,15 @@
     "generate-docs": "api-documenter markdown --input-folder temp --output-folder docs && rm -rf temp",
     "generate-notices": "generate-license-file --input package.json --output ./THIRD-PARTY-NOTICES --overwrite",
     "prepublishOnly": "npm run build",
-    "test": "jest --coverage",
     "storybook": "start-storybook -p 6006",
-    "build-storybook": "build-storybook"
-  },
+    "build-storybook": "build-storybook",
+    "test": "jest --projects tests/jest.unit.config.js --coverage",
+    "coverage:visual": "jest --projects tests/jest.visual.config.js --coverage",
+    "coverage:clean": "rm -rf .nyc_output && rm -rf coverage",
+    "coverage:merge": "istanbul-merge --out coverage/merged/coverage-final.json ./coverage/unit/coverage-final.json  ./coverage/visual-regression/coverage-final.json",
+    "coverage:merge-report": "nyc report --reporter=lcov --reporter=text --temp-dir=./coverage/merged --report-dir=./coverage/merged",
```

Comment:
For my edification what is coverage:merge-report doing? Which lcov files is it merging together?

Reply:
merge-report is only displaying the coverage on screen with the merge json file generated with the merge command